packages/cli/src/commands/start.ts

```diff
@@ -82,26 +82,24 @@ async function askQuestions({ template: defaultTemplate }: Pick<StartOptions, "t
 
   // Ask for project directory
   {
-    const projectRelativePath = await p.text({
+    const projectPath = await p.text({
       message: "Enter the project directory path:",
       placeholder: "./my-agentica-project",
       validate(value) {
-        if (value === "") {
+        if (value.trim() === "") {
           return "Please enter a directory path";
         }
-        if (value[0] !== ".") {
-          return "Please enter a relative path.";
-        }
         if (existsSync(value)) {
           return "Directory already exists";
         }
         return undefined;
       },
     });
-    if (p.isCancel(projectRelativePath)) {
+    if (p.isCancel(projectPath)) {
       process.exit(0);
     }
-    const projectAbsolutePath = join(process.cwd(), projectRelativePath);
+
+    const projectAbsolutePath = join(process.cwd(), projectPath.trim());
```

Comment:
@8471919 
Thank you for your work!
Actually when we accept the absolute path, we need to check whether the path is absolute or not by `isAbsolute` function.
If it is an absolute path, we cannot use `join`